cfgov/v1/jinja2tags.py

```diff
@@ -1,5 +1,7 @@
+from jinja2 import contextfunction
```

Comment:
@Scotchester The backend lint is failing here because we never use it. Can I remove this import?

Reply:
Whoops, yeah, kill it.